que_.py

```diff
@@ -10,14 +10,14 @@ def enqueue(self, value):
     def dequeue(self):
             try:
                 return self.dll.shift()
-            except:
+            except ValueError:
                 raise ValueError("Dequeue does not work on an empty queue")
 
     def peek(self):
         try: 
             return self.dll.tail.prev.value
```

Comment:
Peek is supposed to show you what value will be the next off the queue.  Don't think it should be tail.prev.

Reply:
Due to my using the push method as enqueue, the tail is the first in and should be the first out to comply with standard operation of a que, so the object previous of the tail is the next object in line (2nd in 2nd out) I know it is probably not how this was meant to be done, but at the moment it is how the code functions, essentially a reversed dll in a few ways

Reply:
I disagree, regardless of you you have used the dll methods, .prev shouldn't be the next one in line. To illistrate.
```
head --> next --> next --> next --> tail
tail --> prev --> prev --> prev --> head
```

Imagine that you are in line at a store. The clerk peeks up to look at the whole line, the next person to come off the line is either the head or the tail, not someone in the middle of the line.

Reply:
Ok that makes sense, due to it being a dll with links both ways, I thought I could essentially use the tail as the front of the line. I have now revised the code so that the order of the que complies better with next and prev logic.

Reply:
You should change it back, because you were correct that you can and should use the tail as the front of the line.  You where only wrong in that you thought the tail itself wasn't the node containing the value peek should provide.  You were doing `self.dll.tail.prev.value`, when it should have been `self.dll.tail.value`